nwb_conversion_tools/datainterfaces/blackrockdatainterface.py

```diff
@@ -0,0 +1,121 @@
+        )
+
+        metadata['Ecephys'] = dict(
+            Device=[dict(
+                name='Device_ecephys',
+                description='no description'
+            )],
+            ElectrodeGroup=[],
+        )
+
+        if self.source_data['nsx_to_load'] != 6:
+            metadata['Ecephys']['LFPElectricalSeries'] = dict()
+        else:
+            metadata['Ecephys']['ElectricalSeries'] = dict()
+
+        return metadata
+
+    def run_conversion(self, nwbfile: NWBFile, metadata: dict = None, use_timestamps: bool = False, 
+                       write_as_lfp: bool = False, return_scaled: bool = False, 
+                       save_path: PathType = None, overwrite: bool = False, stub_test: bool = False):
+        """
+        Primary function for converting recording extractor data to nwb.
+
+        Parameters
+        ----------
+        nwbfile: NWBFile
+            nwb file to which the recording information is to be added
+        metadata: dict
+            metadata info for constructing the nwb file (optional).
+            Should be of the format
+                metadata['Ecephys']['ElectricalSeries'] = {'name': my_name,
+                                                           'description': my_description}
+        use_timestamps: bool
+            If True, the timestamps are saved to the nwb file using recording.frame_to_time(). If False (defualut),
+            the sampling rate is used.
+        write_as_lfp: bool (optional, defaults to False)
+            If True, writes the traces under a processing LFP module in the NWBFile instead of acquisition.
+        save_path: PathType
+            Required if an nwbfile is not passed. Must be the path to the nwbfile
+            being appended, otherwise one is created and written.
+        overwrite: bool
+            If using save_path, whether or not to overwrite the NWBFile if it already exists.
+        stub_test: bool, optional (default False)
+            If True, will truncate the data to run the conversion faster and take up less memory.
+        """
+        if self.source_data['nsx_to_load'] != 6:
+            write_as_lfp = True
+
+        super().run_conversion(
+            nwbfile=nwbfile, 
+            metadata=metadata, 
+            use_timestamps=use_timestamps, 
+            write_as_lfp=write_as_lfp, 
+            return_scaled=return_scaled,
+            save_path=save_path, 
+            overwrite=overwrite,
+            stub_test=stub_test
+        )
+
+
+class BlackrockSortingExtractorInterface(BaseSortingExtractorInterface):
+    """Primary data interface class for converting Blackrock spiking data."""
+
+    SX = se.BlackrockSortingExtractor
+
+    def __init__(self, filename: PathType, nsx_to_load: Optional[int] = None):
+        super().__init__(filename=filename, nsx_to_load=nsx_to_load)
+
+    def get_metadata(self):
+        """Auto-populates spiking unit metadata."""
+        metadata = super().get_metadata()
+        return metadata
```

Comment:
This seems like it should not be necessary

Reply:
@luiztauffer this still seems unnecessary

Reply:
you're right, I'll remove it